packages/debugger/lib/data/selectors/index.js

```diff
@@ -458,6 +469,14 @@ const data = createSelectorTree({
     userDefinedTypes: createLeaf(
       ["/state"],
       state => state.info.userDefinedTypes
+    ),
+
+    /**
+     * data.info.eventsAndErrors
+     */
+    eventsAndErrors: createLeaf(
```

Comment:
Hm, good question.

So, what these have in common is that they are contract outputs that are potentially ambiguous, and which we want to use the debugger AST info to disambiguate.

This doesn't apply to return values -- the other sort of contract outputs -- because how to decode those is determined by the external information that is which function was called, rather than by information internal to the output itself (in the form of a selector).

So, I guess you could say what they have in common is that they're contract outputs that have selectors.  I guess we could call that, like, `taggedOutputs` or something?  Does that sound good?

Reply:
`taggedOutputs` sounds good